### DISCUSSION TITLE
Unable to access/view the projects I am admin of

### DISCUSSION THREAD

**Discussion by sainiManveer** [community/community#132197] [Discussions]
### Select Topic Area

Question

### Body

https://github.com/orgs/community/discussions/129860#discussioncomment-9901691
Can anyone help me with this issue?
I am an admin of the various projects but am unable to even view the project and the boards. Even in the settings/access it says I am the admin, but regardless I am unable to access it after migrating my projects to new boards/versions. 
This is happening with multiple projects. I was able to access and do everything as admin in old version, but in new one, I can't even see things.

**Comment by LiteBrite82** [comment]
Hi @sainiManveer 

Thank you for participating in the community! 

Please review our [Code of Conduct](https://github.com/community/community/blob/032a098c5efe572b2d4b2658f4a42f8a56c030ed/CODE_OF_CONDUCT.md) for best practices concerning crossposting. 

I'm closing and locking this discussion as it is a cross post; though I did post some questions and insights on your original discussion. I am excited to continue helping out there.
